client/post-editor/media-modal/index.jsx

```diff
@@ -253,7 +255,10 @@ export class EditorMediaModal extends Component {
 				fileName: fileName,
 				fileContents: blob,
 				mimeType: mimeType
-			}
+			},
+			// conditionally add the height and width of the transient image
+			...width && { width },
+			...height && { height }
```

Comment:
No problems. I rely on these assignments for their terseness rather than their readability. I'll switch to conditional + `Object.assign()`